packages/image_picker/image_picker_ios/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -16,31 +16,24 @@
 #import "FLTImagePickerMetaDataUtil.h"
 #import "FLTImagePickerPhotoAssetUtil.h"
 #import "FLTPHPickerSaveImageToPathOperation.h"
+#import "messages.g.h"
 
-/**
- * Returns the value for the given key in 'dict', or nil if the value is
- * NSNull.
- */
-id GetNullableValueForKey(NSDictionary *dict, NSString *key) {
-  id value = dict[key];
-  return value == [NSNull null] ? nil : value;
+@implementation FLTImagePickerMethodCallContext
+- (instancetype)initWithResult:(nonnull FlutterResultAdapter)result {
+  if (self = [super init]) {
+    _result = [result copy];
+  }
+  return self;
 }
+@end
+
+#pragma mark -
```

Comment:
Are we intentionally leaving an empty string for pragma mark?

Reply:
`#pragma mark -` is a line separator
<img width="182" alt="Screen Shot 2022-04-22 at 10 54 31 AM" src="https://user-images.githubusercontent.com/682784/164768692-93a68cb2-108b-436b-970b-be6b30d7b818.png">

Reply:
In particular, it's useful for cases like this one (a separator between two completely different classes in the same file) where a title for the new section would be redundant, but the visual separation is still helpful.